### DISCUSSION TITLE
Redirect from action table to another resource

### DISCUSSION THREAD

**Discussion by gintoki05** [filamentphp/filament#12478] [Help]
### Package

Table builder

### Package Version

V3.0.0

### How can we help you?

Can someone help me, when i click a action button, i want to redirect to another resource page (but i dont know how to access the route), also send data to the resource, what should i use, action() or url () and how

**Comment by leandrocfe** [comment]
Use Action with url
https://filamentphp.com/docs/3.x/tables/actions#row-actions

**Reply by gintoki05** [reply]
but in the example 

`Action::make('edit')
    ->url(fn (Post $record): string => route('posts.edit', $record))
    `
    
use route that direct to custom view. What i want, its redirect to ListResourcePage, when i change 'posts.edit' to '/admin/listroute', it said route not define. 

`->actions([
                Tables\Actions\Action::make('Terima')
                    ->requiresConfirmation()
                    ->icon('heroicon-m-clipboard-document-check')
                    ->color('info')
                    ->url(function (PendaftaranRajal $record) {
                        $data = [
                            'pendaftaran_rajal_id' => $record->id,
                        ];
                        KunjunganRajal::create($data);
                        route(ListKunjunganRajals::getUrl());
}),`

**Comment by gintoki05** [comment]
i think its dead community haha
